nodeup/pkg/model/kube_scheduler.go

```diff
@@ -196,10 +196,6 @@ func (b *KubeSchedulerBuilder) buildPod(kubeScheduler *kops.KubeSchedulerConfig)
 		flags = append(flags, "--"+flag+"kubeconfig="+kubescheduler.KubeConfigPath)
 	}
 
-	if fi.ValueOf(kubeScheduler.UsePolicyConfigMap) {
```

Comment:
Do we want to continue to support this before 1.23?

Reply:
We don't support K8s 1.23 anymore :)